theme.json.sample

```diff
@@ -4,11 +4,9 @@
   "author": "Mayke Freitas",
   "folder_name": "elegance",
   "development": {
-    "url": "http://demo.edools-dev.com:3000",
+    "url": "http://demo.edools-dev.com:3010",
```

Comment:
Essa mudança força todos que mexem nesse cara a usar o docker.
Essa já é uma cultura alinhada com os afetados por essa mudança?